src/components/pages/Categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <h1>Hello Form Categories</h1>;
```

Comment:
- [ ] Please as demanded by project requirements, kindly add a button with the text `check status` to the categories component.